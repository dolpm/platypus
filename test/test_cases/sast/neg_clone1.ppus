pipe main [] |> unit {
    int x = 1;
    bool b = %x;
    
    |> ();
}