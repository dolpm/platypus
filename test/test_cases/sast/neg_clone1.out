Fatal error: exception Failure("illegal assignment bool = int in bool b = %x;\n")
